### PR TITLE
[SYCL][ESIMD] Test fix for proper intrinsic generation

### PR DESCRIPTION
**Pull request by fineg74**

### CODE REVIEW

**Comment by fineg74** [comment]
Complementary compiler PR https://github.com/intel/llvm/pull/7026

File: SYCL/ESIMD/regression/iselect.cpp [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+//
+//===----------------------------------------------------------------------===//
+// This is basic test for testing proper intrinsics generation.
+
+// REQUIRES: gpu
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl -fsycl-device-only -Xclang -emit-llvm -o %t.comp.ll %s
+// RUN: sycl-post-link -ir-output-only -lower-esimd -S %t.comp.ll -o %t.out.ll
+// RUN: FileCheck --input-file=%t.out.ll %s
+
```
Comment (kbobrovs) [inline SYCL/ESIMD/regression/iselect.cpp:14]: Please make this test runnable

**Review by kbobrovs — APPROVED** [review]

**Comment by fineg74** [comment]
Multiple test failures under Windows only not related to the test from PR

**Comment by kbobrovs** [comment]
> Multiple test failures under Windows only not related to the test from PR

Please list failures and describe problem in PR comments next time.

**Pull request merged**